src/core/webidl.js

```diff
@@ -351,11 +351,11 @@ async function loadStyle() {
 }
 
 export async function run() {
-  const idls = document.querySelectorAll("pre.idl");
-  if (!idls.length) {
-    return;
-  }
-  if (!document.querySelector(".idl:not(pre)")) {
+  const idls = document.querySelectorAll("pre.idl, pre.webidl");
+  if (
+    !document.querySelector(".idl:not(pre)") ||
```

Comment:
Similarly here:
``` js
!document.querySelector(".idl:not(pre), .webidl:not(pre)")
```
This is a good read on combining CSS selectors: https://codenart.github.io/css/2017/08/24/css-9-combining-selectors.html